modules/ossm-accessing-bookinfo-application-using-istio-gateway-injection.adoc

```diff
@@ -0,0 +1,49 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-accessing-bookinfo-application-using-istio-gateway-injection"]
+= Accessing the Bookinfo application by using Istio gateway injection
+:context: ossm-accessing-bookinfo-application-using-istio-gateway-injection
+
+Gateway injection uses the same mechanisms as {istio} sidecar injection to create a gateway from a `Deployment` resource that is paired with a `Service` resource. The `Service` resource can be made accessible from outside an {ocp-product-title} cluster.
+
+.Procedure
+
+. Create the `istio-ingressgateway` deployment and service by running the following command at the CLI:
++
+[source,terminal]
+----
+$ oc apply -n istio-system -f ingress-gateway.yaml
```

Comment:
So at the top we say

> As a security best-practice, Ingress and Egress Gateways should be deployed in a different namespace than the namespace that contains the control plane.

And then here we're creating the deployment and service in the `istio-system` namespace. I'm actually not sure if the above recommendation is for _just_ the istio `Gateway` / `VirtualService` resources created in the bookinfo namespace below or if it's for both the istio resources _and_ the underlying kube resources to be in a different namespace than the controlplane.

Reply:
@luksa when you are back online can you comment on this for me?

Reply:
For this example, it's probably best to deploy the ingress gateway in the `bookinfo` namespace.

Reply:
Good point Nick! Thanks for confirming for me, Marko.

@nrfox I also changed line 46 that syntax to expose the gateway to the cluster. Please let me know if that should be changed back.

oc expose service istio-ingressgateway -n bookinfo